common/protos-solidity/common/asset_locks.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
```

Comment:
This file should not be checked in, but instead should be generated from the file in `protos folder`